<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewsBlurAccountDetails</class>
 <widget class="QWidget" name="NewsBlurAccountDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>281</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="m_lblTitle">
     <property name="text">
      <string>URL</string>
     </property>
     <property name="buddy">
      <cstring>m_txtUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEditWithStatus" name="m_txtUrl" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="m_cbDownloadOnlyUnreadMessages">
     <property name="text">
      <string>Download unread articles only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_lblLimitMessagesShort">
       <property name="text">
        <string>Only download newest X articles per feed</string>
       </property>
       <property name="buddy">
        <cstring>m_spinLimitMessages</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MessageCountSpinBox" name="m_spinLimitMessages"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="HelpSpoiler" name="m_lblLimitMessages" native="true"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="m_gbAuth">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Username</string>
        </property>
        <property name="buddy">
         <cstring>m_txtUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWithStatus" name="m_txtUsername" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="buddy">
         <cstring>m_txtPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWithStatus" name="m_txtPassword" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="m_btnTestSetup">
       <property name="text">
        <string>&amp;Test setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabelWithStatus" name="m_lblTestResult" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>409</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelWithStatus</class>
   <extends>QWidget</extends>
   <header>labelwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithStatus</class>
   <extends>QWidget</extends>
   <header>lineeditwithstatus.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessageCountSpinBox</class>
   <extends>QSpinBox</extends>
   <header>messagecountspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>HelpSpoiler</class>
   <extends>QWidget</extends>
   <header>helpspoiler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_cbDownloadOnlyUnreadMessages</tabstop>
  <tabstop>m_spinLimitMessages</tabstop>
  <tabstop>m_btnTestSetup</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
